code/controllers/subsystems/skybox.dm

```diff
@@ -4,12 +4,13 @@ SUBSYSTEM_DEF(skybox)
 	name = "Space skybox"
 	init_order = INIT_ORDER_SKYBOX
 	flags = SS_NO_FIRE
-	var/list/skybox_cache = list()
+	var/static/list/skybox_cache = list()
```

Comment:
Will this prevent it from being updated when events trigger skybox changes?

Reply:
`static` is an alias for `global` in BYOND iirc